Data/ODBC/testsuite/src/ODBCSQLServerTest.cpp

```diff
@@ -417,6 +417,30 @@ catch(Poco::Exception& ex)
 }
 }
 
+void ODBCSQLServerTest::testUUIDsBulk()
+{
+	try
+	{
+		if (!_pSession) fail("Test not available.");
+
+		_pSession->setFeature("autoBind", true);
+		_pSession->setFeature("autoExtract", true);
+
+		recreateUUIDsTable();
```

Comment:
where exactly is this function? CI does not build or run ODBC tests, you have to do it yourself and make sure it works as intended